### PR TITLE
fixes #7970 fix(nimbus): update prefers reduced motion targeting code

### LINKED ISSUE #7970: Update prefers reduced motion targeting
This issue is to update targeting for users who don't have prefers-reduced-motion enabled, to be more in line with the existing one-click targeting in order to avoid targeting users who have already set Firefox as their default browser. There is a [bugzilla tracking bug here](https://bugzilla.mozilla.org/show_bug.cgi?id=1800128).

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-2940)

### PR DESCRIPTION
**Pull request by jwayn**
Because

* targeting code needs to exclude users who have set firefox to default to align with experiment

This commit

* update prefers reduced motion targeting code to align with experiment requirements

### CODE REVIEW

**Comment by jwayn** [comment]
@jaredlockhart Can we please hold off on approving & merging this one until we get a contextual review from @punamdahiya, who is running the experiment? Thank you!

File: app/experimenter/targeting/constants.py
Comment (punamdahiya) [inline app/experimenter/targeting/constants.py:116]: nit: should it be suffixed ..PREFER_MOTION opposite of REDUCED_MOTION

File: app/experimenter/targeting/constants.py
Comment (punamdahiya) [inline app/experimenter/targeting/constants.py:136]: I think check above should be same as targeting and should remove doesAppNeedPin, we can use this patch to fix desktop_telemetry clause in Line 108 and 109 which incorrectly includes doesAppNeedPin and leaves out new_profile

File: app/experimenter/targeting/constants.py
Comment (punamdahiya) [inline app/experimenter/targeting/constants.py:119]: Above name is used in drop down and getting longer and can be shortened to         "First start-up users on Windows 10 1903 (build 18362) and no prefer reduced motion"

File: app/experimenter/targeting/constants.py
Comment (punamdahiya) [inline app/experimenter/targeting/constants.py:128]: Can we have  FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.targeting  similar to FIRSTRUN_targeting on Line 102 to replace `{first_run} && os.windowsBuildNumber >= 18362 && {new_profile} && "
        "{need_default} `

File: app/experimenter/targeting/constants.py
Comment (punamdahiya) [inline app/experimenter/targeting/constants.py:128]: noticed `first_run= FIRST_RUN.targeting probably` in above targetings, probably ok but will be good to run with @jaredlockhart  if its encouraged to use unique variable name between targetings

**Review by punamdahiya — APPROVED** [review]

File: app/experimenter/targeting/constants.py
Comment (jaredlockhart) [reply app/experimenter/targeting/constants.py:128]: @punamdahiya can you give me an example of what you mean?

**Comment by jaredlockhart** [comment]
@jwayn Looks like @punamdahiya has approved, is this ok to merge?

File: app/experimenter/targeting/constants.py
Comment (jwayn) [reply app/experimenter/targeting/constants.py:128]: @jaredlockhart I think Punam is highlighting that the `first_run` is reused as a format variable amongst multiple configuration declarations, and may be confusing for some.

**Comment by jwayn** [comment]
@jaredlockhart I think this is good to go!

File: app/experimenter/targeting/constants.py
Comment (jaredlockhart) [reply app/experimenter/targeting/constants.py:128]: Ah interesting point, I'll think about it 👍

File: app/experimenter/targeting/constants.py
Comment (jaredlockhart) [inline app/experimenter/targeting/constants.py:775]: Oh let's leave this in here.  We'd need a more detailed plan for deprecating an existing targeting config.

**Review by jaredlockhart — CHANGES_REQUESTED** [review]

File: app/experimenter/targeting/constants.py
Comment (jwayn) [reply app/experimenter/targeting/constants.py:775]: I just added this a few days ago with [issue 7959](https://github.com/mozilla/experimenter/issues/7959), and it isn't in use as far as I'm aware. I'm happy to leave it in if you think it will be of use though!

Comment (jaredlockhart) [reply app/experimenter/targeting/constants.py:775]: Oh! Okay then yeah, should be fine.  Thanks!

**Review by jaredlockhart — APPROVED** [review]

**Pull request merged**
